### DISCUSSION TITLE
[远期] 自测

### DISCUSSION THREAD

**Discussion by Guanchishan** [MindongLab/yngdieng#21] [产品功能 ideas]
**你提出的功能想解决什么问题**

- 规律性地记忆特定内容

**你觉得理想的解决方案是什么**

1. 以[单词表](https://github.com/MindongLab/yngdieng/issues/20)为内容单位，允许用户自测
2. 自测方式类似网上的背单词软件

**其他可能的解决方案**

- 见其他信息第2条。

**其他信息**

1. 如果单词表功能真的[支持收藏网页](#20 )的话，考虑到网页并不能机器友好地提供单词信息，所以可以跳过单词表内的网页内容并与用户说明；这样的话可能可以允许用户自行补充词汇并自由选择是否提供给我们
2. 如果可能还可以允许输出为Anki卡牌，但这样可能导致词汇数据泄露（如果在乎的话。毕竟未来如果榕典有出错并追加订正的话，并不能保证挪用数据者及时跟进，从而遏制错误传播）

**Comment by ztl8702** [comment]
（相关）在google 搜英文的时候看到这个word coach功能：

<img width="425" alt="Screen Shot 2020-03-20 at 9 35 17 pm" src="https://user-images.githubusercontent.com/2714446/77168386-b20c4580-6af2-11ea-8eae-6630508baea2.png">

<img width="415" alt="Screen Shot 2020-03-20 at 9 35 59 pm" src="https://user-images.githubusercontent.com/2714446/77168450-d2d49b00-6af2-11ea-8f7e-0df2542e14bb.png">

<img width="385" alt="Screen Shot 2020-03-20 at 9 35 28 pm" src="https://user-images.githubusercontent.com/2714446/77168420-c3555200-6af2-11ea-880e-533f0edeefc9.png">


不过这个更像是“词义辨析”和你说的记忆曲线的东西还是不太一样

**Comment by Guanchishan** [comment]
词义辨析蛮有意思的，用图片来辨是个亮点。不过这个辨析真需要对福州话有很深的把握，可能要请老母语者来帮忙

**Comment by ztl8702** [comment]
其实每周线上学习就有提到不少词汇的辨析。要去整理一下。

On Fri, 20 Mar 2020 at 22:53, Davidzdh <notifications@github.com<mailto:notifications@github.com>> wrote:

词义辨析蛮有意思的，用图片来辨是个亮点。不过这个辨析真需要对福州话有很深的把握，可能要请老母语者来帮忙
